### PR TITLE
feat: annunciator lights on checklist

### LINKED ISSUE #5138: Include ANN LT TEST in the checklist
### Aircraft Version

Development

### Description

The ANN LT TEST should be included in the MSFS checklist.

This issue is created to split up the remaining items in #528 so they can be individually tracked.

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

SjotgunSjonnie#9623

### PR DESCRIPTION
**Pull request by oim1**
Fixes #5138 

## Summary of Changes
Added Annunciator Lights Check/As Required to the interactive checklist.

## Additional context

Discord username: oim

## Testing instructions
Load up the sim and check if Annunciator Lights is in the checklist under cockpit preparation. Press the eye button on the right and it should highlight the switch. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 <!--  1. [Changed Area] Title of changes - @github username (Name)  -->
 
 ## 0.7.0
+1. [MISC] Added annunciator lights to interactive checklist - @oim1 (oim)
```
Comment (davidwalschots) [inline .github/CHANGELOG.md:8]: Please append to the changelog instead of prepending to it.

**Review by davidwalschots — COMMENTED** [review]
Thanks for your first contribution!

**Review by davidwalschots — APPROVED** [review]

**Comment by davidwalschots** [comment]
I'm not sure what you're doing, but your workflow seems a bit unusual, having so many commits for a small change.
Please spend a bit more time to understand and improve your usage of Git for any future PR. Thanks! 😀

**Comment by 2hwk** [comment]
@oim1 Merged for compatibility with SU5, please pull changes.

**Comment by SkylarHutter** [comment]
Quality Assurance Tester/Trainee Report

Object of testing: #5414
Tier of Testing: 1
Date: 28/07/2021

Testing Process: Loaded up in sim, checklist for Annunciator, eye button correctly lit up as expected.
Testing Results: Passed

**Comment by ShadowFrost3** [comment]
Quality Assurance Tester/Trainee Report

Object of testing: #5414
Tier of Testing: 1
Date: 28/07/2021

Testing Process: Cold and dark a320 load, open in game checklist, open overhead panel, select the eye beside "annunciator lights" and watch to see if it highlights annunciator light switch. 
Negatives: N/A
Testing Results: Passed

image

**Pull request merged**